<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMTP</class>
 <widget class="QWidget" name="SMTP">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Button_save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_break">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Server">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>smtp.163.com</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_UserName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_SenderMail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_ACode">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="Input_Content">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;你好呀宝贝&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_ReceiveEmail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Title">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>这是一次试验</string>
   </property>
  </widget>
  <widget class="QLabel" name="server">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱服务器</string>
   </property>
  </widget>
  <widget class="QLabel" name="SenderEmailAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发件人邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="AuthorizationCode">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>授权码</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReceiverEmailAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>收件人邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="Content">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Send">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Output_status">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
